custom_components/frigate/binary_sensor.py

```diff
@@ -92,6 +95,14 @@ def device_info(self) -> dict[str, Any]:
             "via_device": get_frigate_device_identifier(self._config_entry),
             "name": get_friendly_name(self._cam_name),
             "model": self._get_model(),
+            "configuration_url": self._config_entry.data.get(
+                CONF_URL, self._config_entry.data.get(CONF_HOST, "")
```

Comment:
`CONF_HOST` was only used in the fairly old v1 of the config entry format, which is since migrated to CONF_URL. Almost no-one will have that set anymore. It would be better to not use CONF_HOST here, as I'll probably rip out all of v1->v2 migration code soon anyway, which will remove all references to CONF_HOST (i.e. lets not add another one).

Reply:
Should I change the tests to not use `CONF_HOST` then?

Reply:
Should I change the tests to not use `CONF_HOST` then?